core/unit_test/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ if(NOT GTest_FOUND) # fallback to internal gtest
   set(GTEST_SOURCE_DIR ${Kokkos_SOURCE_DIR}/tpls/gtest)
 
   #need here for tribits
-  kokkos_include_directories(${GTEST_SOURCE_DIR})
+  kokkos_include_directories(SYSTEM ${GTEST_SOURCE_DIR})
```

Comment:
`gcc 8` warns inside `gtest.h`. I don't want to modify a TPL.

If we make the `gtest` directory a system include directory, compilers should not warn about it.